### DISCUSSION TITLE
add husk to support auto prettier --write on git commit

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4242] [Feedback]
now it is diffucute to modify code in github website when modify some small code 

often  be prevent by `npm run test: prettier -- test `

or remove ` prettier -- test `

**Comment by rtivital** [comment]
1. Why do you think that we should merge code that does not meet codebase standards?
2. Why do you think that contributing code features via github website is a good idea?

**Reply by vaynevayne** [reply]
1. run prettier in git commit hook 
2. For small changes, it's simple.
